admin-dev/themes/new-theme/js/app/components/checkbox.vue

```diff
@@ -0,0 +1,92 @@
+<!--**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *-->
+<template>
+  <div class="md-checkbox md-checkbox-inline">
+    <label>
+      <input
+        v-if="Array.isArray(checked)"
+        type="checkbox"
+        :checked="checked.includes(value)"
+        :class="classes"
+        :disabled="disabled"
+        @change="change"
+      >
+      <input
+        v-else
+        type="checkbox"
+        :checked="checked"
+        :class="classes"
+        :disabled="disabled"
+        @change="$emit('input', $event.target.checked)"
+      >
+
+      <slot>
+        <!-- - Fallback content -->
+        <i class="md-checkbox-control" />
+      </slot>
+    </label>
+  </div>
+</template>
```

Comment:
I'm wondering if `admin-dev/themes/new-theme/js/app/components/checkbox.vue` is the right location for this? When we consider other Vue files in `admin-dev/themes/new-theme/js/app/components/`